bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/Util.java

```diff
@@ -33,13 +33,7 @@ public class Util {
      * @return true or false
      */
     public static boolean inRange(int value, int min, int max) {
-        if (value < min) {
-            return false;
-        }
-        if (value > max) {
-            return false;
-        }
-        return true;
+        return !((value < min) || (value > max));
```

Comment:
Consider if:
```suggestion
        return value >= min && value <= max;
```

would be more readable.